src/main/java/com/rultor/agents/IndexesRequests.java

```diff
@@ -39,8 +47,106 @@
  * @version $Id$
  */
 public final class IndexesRequests implements SuperAgent {
+    /**
+     * Xpath expression for retrieving log entries from a talk's XML.
+     */
+    private static final String ARCHIVE_LOG = "//archive/log";
```

Comment:
DON'T use constants in order to solve "duplicate literals" problems. constants are not supposed to do this work. constants exist in Java in order to declare an object that should live through the entire life of the application. what object is "ARCHIVE_LOG"? it is not an object, but a supplementary piece of text used in one of your methods.